datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -536,3 +544,440 @@ def inspect_for_age(metadata, values, config):  # noqa: C901
     confidence_level = np.round(confidence_level, 2)
 
     return confidence_level, debug_info
+
+
+def inspect_for_iban(metadata, values, config):  # noqa: C901
+    prediction_factors_weights = config[PREDICTION_FACTORS_AND_WEIGHTS]
+    debug_info: Dict[str, Any] = {}
+
+    # Value Logic
+    if prediction_factors_weights.get(VALUES, 0) > 0:
+        values_score = 0.0
+        try:
+            if config[VALUES][PREDICTION_TYPE] == "regex":
+                values_score = match_regex_for_values(values, config[VALUES][REGEX])
+            elif config[VALUES][PREDICTION_TYPE] == "library":
+                iban_score = 0.0
+                for val in values:
+                    try:
+                        if IBAN(val, allow_invalid=True).is_valid:
+                            iban_score += 1.0
+                    except Exception:
+                        pass
+                values_score = iban_score / len(values)
+            else:
+                raise Exception(
+                    "Inappropriate values_prediction_type %s"
```

Comment:
use f-strings (here and elsewhere)

Reply:
Used f-string at all the places instead of "%s" formatting.